packages/flutter/lib/src/material/expansion_panel.dart

```diff
@@ -351,6 +355,13 @@ class ExpansionPanelList extends StatefulWidget {
   /// constructor.)
   final Object initialOpenPanelValue;
 
+  /// Padding is applied to panel header when expanded.
+  /// By default, some space is added to the header vertically above and below
+  /// during expansion.
+  /// This property can be used to adjust this behavior, for example to achieve
```

Comment:
I think this is self explanatory from the first sentence. Perhaps remove it altogether?

Reply:
Yep, removed.